apps/dashboard/app/views/projects/import.html.erb

```diff
@@ -34,6 +34,17 @@
                                 required: false,
                                 data: { 'path-selector': true })%>
 
+              <div class="dropdown mt-2">
+                <button class="btn btn-secondary dropdown-toggle w-100" type="button" id="directory-dropdown" data-bs-toggle="dropdown" aria-expanded="false">
+                  <%= I18n.t('dashboard.jobs_select_directory_placeholder') %>
+                </button>
+                <ul class="dropdown-menu w-100" aria-labelledby="directory-dropdown" id="directory-dropdown-menu">
+                  <% @directories.each do |dir| %>
+                    <li><a class="dropdown-item" href="javascript:void(0);" data-value="<%= dir %>"><%= dir %></a></li>
+                  <% end %>
+                </ul>
+              </div>
```

Comment:
Seems like this should be a part of the form. We need to be careful with HTML semantics for screen readers and that includes javascript manipulation.

Seems like this should all be a part of the `form` so a screen reader user isn't confused on what's what.

Reply:
I will fix this today.